microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -45,136 +46,140 @@ This article uses five demo webpages, all sourced at https://github.com/Microsof
--->
+1. Open the webpage you want to analyze. For example, open the [Scattered objects](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-03.html) demo page in a new window or tab.
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
 
-1. In DevTools, open the **Memory** tool.
+1. In DevTools, on the **Activity Bar**, select the **Memory** tab.  If that tab isn't visible, click the **More tools** (![More tools icon](./heap-snapshots-images/more-tools-icon.png)) button.
 
 1. In the **Select profiling type** section, select the **Heap snapshot** option button.
 
-1. Click the **Take snapshot** button, and then click **Start**.  Or, press **Ctrl+E** (Windows, Linux) or **Command+E** (macOS).
+1. Under **Select JavaScript VM instance**, select the process that you want to profile.
 
-![Selecting the 'Heap snapshot' profiling type in the Memory tool](./heap-snapshots-images/heap-snapshots.png)
+1. Click the **Take snapshot** button:
 
-**Snapshots** are initially stored in the renderer process memory.  Snapshots are transferred to the DevTools on demand, when you click the snapshot icon to view it.
+  ![The Memory tool, the Heap snapshot option is selected, and the Take snapshot button is highlighted](./heap-snapshots-images/heap-snapshots.png)
 
-After the snapshot has been loaded into DevTools and has been parsed, the number below the snapshot title appears and shows the [total size of the reachable JavaScript objects](memory-101.md#object-sizes).
+After the newly recorded heap snapshot has been loaded into DevTools and has been parsed, the snapshot is displayed and a new entry appears in the **Profiles** sidebar under **HEAP SNAPSHOTS**:
 
 ![Total size of reachable objects](./heap-snapshots-images/heap-snapshots-all.png)
 
-> [!NOTE]
-> Only reachable objects are included in snapshots.  Also, taking a snapshot always starts with a garbage collection.
+The number below the new sidebar item shows the total size of the reachable JavaScript objects. To learn more about object sizes in the heap snapshot, see [Object sizes and distances](./memory-101.md#object-sizes-and-distances) in _Memory terminology_.
 
+The snapshot only displays the objects from the memory graph that are reachable from the global object. Taking a snapshot always starts with a garbage collection.
 
-<!-- ====================================================================== -->
-## Clear snapshots
 
-Click the **Clear all profiles** icon to remove snapshots (both from DevTools and any memory associated with the renderer process).
+<!-- ------------------------------ -->
+#### Take another snapshot
 
-![Remove snapshots](./heap-snapshots-images/heap-snapshots-all-hover-clear-all-profiles.png)
+To take another snapshot when one is already displayed in the **Memory** tool, click **Profiles** above the existing snapshot in the sidebar:
```

Comment:
```suggestion
To take another snapshot when one is already displayed in the **Memory** tool, in the sidebar, click **Profiles** above the existing snapshot:
```
i list steps in order of eye scanning.  First, find X with your eye.  Then find Y in the UI with your eye. ...